examples/nextjs-14-permit/src/app/api/permissions/route.ts

```diff
@@ -1,74 +1,39 @@
 import { NextResponse } from "next/server";
-import arcjet, {
-  detectBot,
-  shield,
-  slidingWindow,
-} from "@arcjet/next";
+import { detectBot, slidingWindow } from "@arcjet/next";
 import { currentUser } from "@clerk/nextjs/server";
-import { Permit } from "permitio";
-
-const aj = arcjet({
-  // Get your site key from https://app.arcjet.com
-  // and set it as an environment variable rather than hard coding.
-  // See: https://nextjs.org/docs/app/building-your-application/configuring/environment-variables
-  key: process.env.ARCJET_KEY!,
-  // Define a global characteristic that we can use to identify users
-  characteristics: ["fingerprint"],
-  rules: [
-    // Shield detects suspicious behavior, such as SQL injection and cross-site
-    // scripting attacks. We want to ru nit on every request
-    shield({
-      mode: "LIVE", // will block requests. Use "DRY_RUN" to log only
-    }),
-    slidingWindow({
-      mode: "LIVE",
-      max: 2,
-      interval: 1,
-    }),
-    detectBot({
-      mode: "LIVE",
-      block: ["AUTOMATED"],
-    }),
-  ],
-});
-
-const permit = new Permit({
-  pdp: process.env.PERMIT_PDP!,
-  token: process.env.PERMIT_TOKEN!,
-});
+import { aj } from "@/lib/arcjet";
+import { permit } from "@/lib/permit";
 
 export async function GET(req: Request) {
   let user = await currentUser();
   if (!user) {
     return NextResponse.json({ canUpdate: false });
   }
 
-  const decision = await aj.protect(req, { fingerprint: user.id });
+  const decision = await aj
+    // Add a sliding window rule to limit the number of requests to 2 per second
+    .withRule(slidingWindow({ mode: "LIVE", max: 2, interval: 1 }))
+    // Add bot detection to block automated requests
+    .withRule(detectBot({ mode: "LIVE", block: ["AUTOMATED"] }))
```

Comment:
If these aren't being configured via some dynamic data, I'd suggest moving them out of the handler. Generally I name the export from `@/lib/arcjet` as `arcjet` and then outside of the handler I do something like:
```
let aj = arcjet
    // Add a sliding window rule to limit the number of requests to 2 per second
    .withRule(slidingWindow({ mode: "LIVE", max: 2, interval: 1 }))
    // Add bot detection to block automated requests
    .withRule(detectBot({ mode: "LIVE", block: ["AUTOMATED"] }));
```

The reason for this is that `withRule` is not zero-cost, so anything static should be done once (not per request).